test/WebJobs.Script.Tests.Integration/ApplicationInsights/ApplicationInsightsTestFixture.cs

```diff
@@ -185,5 +188,19 @@ protected override ITelemetryChannel CreateTelemetryChannel()
                 return channel;
             }
         }
+
+        private class TestScriptHostConfigReader : DefaultScriptHostConfigReader
+        {
+            public override string Read(string hostJsonPath)
+            {
+                string hostJson = base.Read(hostJsonPath);
+
+                // Add a custom 'functions' section to keep the tests simpler
```

Comment:
Modifying host.json to white-list the "Scenarios" function -- but without writing to disk. Modifying the host.json was my initial approach but it needs to be undone to prevent other tests from failing.